packages/@tailwindcss-postcss/src/__snapshots__/index.test.ts.snap

```diff
@@ -393,6 +393,11 @@ exports[`\`@import 'tailwindcss'\` is replaced with the generated CSS 1`] = `
     --line-height-8: 2rem;
     --line-height-9: 2.25rem;
     --line-height-10: 2.5rem;
+    --perspective-dramatic: 100px;
+    --perspective-near: 300px;
+    --perspective-normal: 500px;
+    --perspective-midrange: 800px;
+    --perspective-distant: 1200px;
```

Comment:
Are these the final names for the default values we ship?

Reply:
Yes, taken from [here](https://github.com/tailwindlabs/tailwindcss/pull/10982#issuecomment-1710739315).